tests/cli/bmccreds/test_main.py

```diff
@@ -160,7 +213,71 @@ def test_random_password_with_domain(self):
             xnames=self.mock_hsm_client.get_and_filter_bmcs.return_value,
             domain='bmc',
             force=False,
-            report_format='pretty'
+            report_format='pretty',
+            length=RANDOM_PASSWORD_LENGTH,
+            allowed_chars=VALID_CHAR_SETS_STRING
+        )
+
+    def test_random_password_with_password_length(self):
+        """Supplying --random-password with a password length"""
+        self.cli_args = ['bmccreds', '--random-password', '--password-length', '8']
```

Comment:
I would suggest changing this to something other than the default. Otherwise, your test can't tell whether it's correctly passing the CLI argument or if it's just mistakenly taking the default.

```suggestion
        self.cli_args = ['bmccreds', '--random-password', '--password-length', '10']
```

Reply:
Addressed